box/Project.toml

```diff
@@ -8,3 +8,6 @@ OrdinaryDiffEq = "1dea7af3-3e70-54e6-95c3-0bf5283fa5ed"
 Random = "9a3f8284-a2c9-5f02-9a11-845980a1fd5c"
 StatsPlots = "f3b207a7-027a-5e70-b257-86293d7955fd"
 Thermodynamics = "b60c26fb-14c3-4610-9d3e-2d17fe7ff00c"
+
+[compat]
+CLIMAParameters = "0.8.6"
```

Comment:
I vote to keep it at "0.8" and not "0.8.6" in our toml files. The parameters are updated very often and we don't want to pin ourselves to some old version.